std/array.d

```diff
@@ -2767,17 +2786,17 @@ if (isDynamicArray!A)
     }
 
     /**
-     * Returns the capacity of the array (the maximum number of elements the
-     * managed array can accommodate before triggering a reallocation).  If any
-     * appending will reallocate, $(D capacity) returns $(D 0).
+     * Returns: the capacity of the array (the maximum number of elements the
+     * managed array can accommodate before triggering a reallocation). If any
+     * appending will reallocate, returns `0`.
      */
     @property size_t capacity() const @safe pure nothrow
     {
         return _data ? _data.capacity : 0;
     }
 
     /**
-     * Returns the managed array.
+     * Returns: The managed array.
```

Comment:
FWIW you use "the" after `Returns:` in the change above. It might make sense to agree on a common convention here as well.